{"report": {"environment": {"Python": "3.12.0", "Platform": "Windows-10-10.0.19045-SP0"}, "tests": [{"name": "test_at.py::test_pow_2", "duration": 0.0014937999994799611, "run_index": 0, "setup": {"name": "setup", "duration": 0.00047139999969658675, "outcome": "passed"}, "call": {"name": "call", "duration": 0.0002862999999706517, "outcome": "passed"}, "teardown": {"name": "teardown", "duration": 0.0002647000001161359, "outcome": "passed"}, "outcome": "passed"}, {"name": "test_at.py::test_pow_2_1", "duration": 0.001115100000788516, "run_index": 1, "setup": {"name": "setup", "duration": 0.00031209999997372506, "outcome": "passed"}, "call": {"name": "call", "duration": 0.0002471000007062685, "outcome": "passed"}, "teardown": {"name": "teardown", "duration": 0.0002438000001347973, "outcome": "passed"}, "outcome": "passed"}, {"name": "test_at.py::test_pow_2_2", "duration": 0.0013989000017318176, "run_index": 2, "setup": {"name": "setup", "duration": 0.00028780000047845533, "outcome": "passed"}, "call": {"name": "call", "duration": 0.00044190000062371837, "outcome": "failed", "longrepr": "def test_pow_2_2():\n>       assert pow_2(22) == 1600\nE       assert 484 == 1600\nE        +  where 484 = pow_2(22)\n\ntest_at.py:15: AssertionError"}, "teardown": {"name": "teardown", "duration": 0.0003814000001511886, "outcome": "passed"}, "outcome": "failed"}], "summary": {"passed": 2, "failed": 1, "num_tests": 3, "duration": 0.09873580932617188}, "created_at": "2024-04-18 19:45:13.406809"}}